core/shared/src/main/scala-2/zio/prelude/Newtype.scala

```diff
@@ -284,7 +284,13 @@ abstract class Newtype[A] extends NewtypeVersionSpecific {
    */
   def wrapAll[F[_]](value: F[A]): F[Type] = macro zio.prelude.Macros.wrapAll_impl[F, A, Type]
 
-  implicit def classTag(implicit underlying: ClassTag[A]): ClassTag[Type] = Newtype.classTag
+  /**
```

Comment:
Shouldn't this rather be a regular comment with `/*` instead of `/**`?

Reply:
Fixed :)